docs/getting-started/linux-standalone.md

```diff
@@ -0,0 +1,130 @@
+# Linux Standalone
+
+All Intel, AMD and ARM systemd-based operating systems are supported including
+CentOS, Debian, Oracle, Red Hat and Ubuntu. DEB and RPM packages are available
+to download at
+[https://github.com/signalfx/splunk-otel-collector/releases
+](https://github.com/signalfx/splunk-otel-collector/releases).
+
+## Getting Started
+
+This distribution comes with [default
+configurations](https://github.com/signalfx/splunk-otel-collector/blob/main/cmd/otelcol/config/collector)
+which require the following environment variables:
+
+- `SPLUNK_REALM` (no default): Which realm to send the data to (for example: `us0`)
+- `SPLUNK_ACCESS_TOKEN` (no default): Access token to authenticate requests
+- `SPLUNK_MEMORY_TOTAL_MIB` (no default): Total memory allocated to the Collector.
+
+<details>
```

Comment:
This details section doesn't seem to be rendering correctly: https://github.com/signalfx/splunk-otel-collector/blob/98b14cb3abd14117f703b6d3e7807d82e2332d36/docs/getting-started/linux-standalone.md

Reply:
Hmm this works for me -- you should be able to select the > icon and it will expand

Reply:
I'm seeing:
![Screen Shot 2021-02-09 at 4 58 43 PM](https://user-images.githubusercontent.com/36963516/107433987-1e6fa900-6af8-11eb-81cd-246245bc8a0e.png)

Reply:
oh, I missed the formatting -- good call...

Reply:
OK, fixed